_pages/en_US/restoring-nand.md

```diff
@@ -49,7 +49,17 @@ It is very important to test that your NAND backup is working before attempting
 8. Click `OK`
 9. Launch any Nintendo DS ROM (`.nds` file)
 
-If no$gba loads the DSi menu, then continue to the next section. If it shows any kind of error ***do not flash that backup***!
+If no$gba loads the DSi menu (or the Unlaunch Filemenu), then continue to the next section. If it shows any kind of error ***do not flash that backup***!
+
+## Uninstalling Unlaunch from your NAND backup (optional)
+Follow this if you dumped your NAND backup after you installed Unlaunch and you would like to uninstall Unlaunch from your system.
+1. Download the latest version of the [Unlaunch installer](https://problemkaputt.de/unlaunch.zip)
+1. Extract `UNLAUNCH.DSI` from `unlaunch.zip`
+1. Launch `UNLAUNCH.DSI` in no$gba and start it from the Game Card slot
+- This should start the Unlaunch installer, which looks similar to to the Unlaunch Filemenu
+4. Choose `Uninstall`
+4. Once complete, choose `Power down`
+4. Launch any Nintendo DS ROM again, and ensure your DSi menu loads and is working properly
```

Comment:
```suggestion
1. Choose `Uninstall`
1. Once complete, choose `Power down`
1. Launch any Nintendo DS ROM again, and ensure your DSi menu loads and is working properly
```
why are these `4.`?

Also ditto on the above comment, while you're not told to do so it's possible someone may close no$gba so I think it should either be explicitly stated not to close it in the last section or mentioned here that you'll need to reconfigure it if you closed it.

Reply:
The preview showed that it reset it back to 1,2,3 because of the bullet, so I gave it 4. Should I just write 1-6? Or did I do something wrong?

Reply:
And, reconfigure it?

Reply:
Ah, I think you need to indent the bullet (after 3)

Also I think jekyll might not even respect the starting number fyi